splunk_eventgen/lib/plugins/generator/jinja.py

```diff
@@ -1,6 +1,8 @@
 from __future__ import division
 from generatorplugin import GeneratorPlugin
-import datetime, time, os
+import datetime
+import time
+import os
 try:
     import ujson as json
```

Comment:
this might not be related to this PR. Just wondering if we have no fall back handler for json module loading, what will happen if the json is referred in the below code?

Reply:
There is a fallback to the generic json library, and if used, you lose the ability to output multiline events.